docs/API.md

```diff
@@ -446,6 +446,20 @@ mysql2
 | http.status_code | The HTTP status code of the response.                     |
 | http.headers.*   | A recorded HTTP header.                                   |
 
+#### sails
+
+Much of the Sails instrumentation is provided by other component instrumentations,
+such as the DNS and HTTP instrumentations, Express, and the various database drivers.
+However, as those instrumentations might cause traces to have a large number of
+superfluous spans, consider selectively enabling the instrumentations desired when
+using Sails.
```

Comment:
This is helpful information but could it be more clear to just state what this instrumentation traces with a lighter note of its relationship w/ Express?